pyhtools/attackers/web/email_scraper.py

```diff
@@ -0,0 +1,74 @@
+from bs4 import BeautifulSoup
+import requests, re, lxml
+from requests import exceptions
+from urllib import parse
+from collections import deque
+from pyhtools.UI.colors import BRIGHT_RED, BRIGHT_WHITE, BRIGHT_YELLOW
+
+##----------------Author: astralm0nke on GitHub-----------------##
+class EmailScraper:
+    USER_URL = str(input('[+] Specify Target URL to Scan: '))
+    USER_COUNT = int(input('[+] Specify Count Limit: '))
+    COUNT = 0
+    URLS = deque([USER_URL])
+    SCRAPPED_URLS = set()
+    EMAILS = set()
```

Comment:
we should avoid using `input` while writing a class instead we can assign these variables in `__init__ ` method and use `argparse` to accept inputs in `__main__` block.

main objective to follow this is others can use pyhtools as per their requirements.